api/inc/svc_exports.h

```diff
@@ -116,6 +116,7 @@
 /* SVC immediate values for hardcoded table (call from unprivileged) */
 #define UVISOR_SVC_ID_UNVIC_OUT        UVISOR_SVC_FIXED_TABLE(0, 0)
 #define UVISOR_SVC_ID_REGISTER_GATEWAY UVISOR_SVC_FIXED_TABLE(3, 0)
+#define UVISOR_SVC_ID_BOX_INIT_NEXT    UVISOR_SVC_FIXED_TABLE(5, 0)
```

Comment:
When we removed the secure gateway handlers we decided not to shift the existing ones, but to leave the unused and then continue with the available slots. We will reuse 1 and 2 when we wrap. The next one to use is 4 (has never been used before).